packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -60,11 +60,9 @@ class ALPNCache {
       ALPNProtocols: ['h2', 'http/1.1'],
       rejectUnauthorized: false,
     }).then(socket => {
-      socket.once('secureConnect', () => {
```

Comment:
`createTLSSocket` is only used for client certificates. Its internally using `createConnectionAsync` which we don't want to modify.  Changing `createTLSSocket` seems reasonable.

Reply:
Sounds good.